packages/perseus/src/widgets/numeric-input.tsx

```diff
@@ -291,12 +284,12 @@ export class NumericInput extends React.Component<Props, State> {
     };
 
     shouldShowExamples: () => boolean = () => {
-        const noFormsAccepted = this.props.answerForms?.length === 0;
+        const noFormsAccepted = this.props.answerForms.length === 0;
         // To check if all answer forms are accepted, we must first
         // find the *names* of all accepted forms, and see if they are
         // all present, ignoring duplicates
         const answerFormNames: ReadonlyArray<string> = _.uniq(
-            this.props.answerForms?.map((form) => form.name),
+            this.props.answerForms.map((form) => form.name),
```

Comment:
These were spotted by `@typescript-eslint/no-unnecessary-condition` so I guess it's not totally useless.